### PR TITLE
[Doc Revamp]1684145 [build tabs] 

### PR DESCRIPTION
**Pull request by v-pmuthyala**

### CODE REVIEW

File: msteams-platform/tabs/what-are-tabs.md [hunk 1]
```diff
@@ -5,14 +5,14 @@ description: An overview of custom tabs on the Teams platform
 ms.topic: overview
 ms.author: lajanuar
 ---
-# What are Microsoft Teams tabs?
+# What are Microsoft Teams tabs
```
Comment (Rajeshwari-v) [inline msteams-platform/tabs/what-are-tabs.md:8]: Avoid questions as heading. You can have - Microsoft Teams tabs as heading

File: msteams-platform/tabs/what-are-tabs.md [hunk 2]
```diff
@@ -5,14 +5,14 @@ description: An overview of custom tabs on the Teams platform
...
 
 Tabs are Teams-aware webpages embedded in Microsoft Teams. They are simple HTML <iframe\> tags that point to domains declared in the app manifest and can be added as part of a channel inside a team, group chat, or personal app for an individual user. You can include custom tabs with your app to embed your own web content in Teams or add Teams-specific functionality to your web content. *See* [Teams JavaScript client SDK](/javascript/api/overview/msteams-client).
 
 > [!NOTE]
-> Chrome 80, scheduled for release in early 2020, introduces new cookie values and imposes cookie policies by default. It's recommended that you set the intended use for your cookies rather than rely on default browser behavior. *See* [SameSite cookie attribute (2020 update)](../resources/samesite-cookie-update.md).
+> Chrome 80, scheduled for release in early 2020, introduces new cookie values and imposes cookie policies by default. It is recommended that you set the intended use for your cookies rather than rely on default browser behavior. *See* [SameSite cookie attribute (2020 update)](../resources/samesite-cookie-update.md).
```
Comment (Rajeshwari-v) [inline msteams-platform/tabs/what-are-tabs.md:13]: ...suggestion
You can set the intended use for your cookies. You need not rely on default browser behavior. For more information on cookies, see [SameSite cookie attribute (2020 update)](../resources/samesite-cookie-update.md).
...

File: msteams-platform/tabs/what-are-tabs.md [hunk 3]
```diff
@@ -5,14 +5,14 @@ description: An overview of custom tabs on the Teams platform
...
 
-There are two types of tabs available in Teams — channel/group and personal. Channel/group tabs deliver content to channels and group chats, and are a great way to create collaborative spaces around dedicated web-based content. Personal tabs, along with personally-scoped bots, are part of personal apps and are scoped to a single user. They can be pinned to the left navigation bar for easy access.
+There are two types of tabs available in Teams — channel or group and personal. Channel or group tabs deliver content to channels and group chats, and are a great way to create collaborative spaces around dedicated web-based content. Personal tabs, along with personally-scoped bots, are part of personal apps and are scoped to a single user. They can be pinned to the left navigation bar for easy access.
 
 ## Lesser known tab features
```
Comment (Rajeshwari-v) [inline msteams-platform/tabs/what-are-tabs.md:17]: Tab features - fits well here.

File: msteams-platform/tabs/what-are-tabs.md [hunk 4]
```diff
@@ -35,15 +35,15 @@ There are two types of tabs available in Teams — channel/group and personal. C
 **Example:** You create personal tabs that provide *about* and *help* webpage content to users.
 
 **Scenario:** Provide access to items that your users interact with regularly for cooperative dialogue and collaboration. \
-**Example:** You create a channel/group tab with deep linking to individual items.
+**Example:** You create a channel or group tab with deep linking to individual items.
```
Comment (Rajeshwari-v) [inline msteams-platform/tabs/what-are-tabs.md:38]: Can we have a table structure here?

File: msteams-platform/tabs/what-are-tabs.md [hunk 5]
```diff
@@ -35,15 +35,15 @@ There are two types of tabs available in Teams — channel/group and personal. C
...
 
 ## How do tabs work?
 
 A custom tab is declared in the app manifest of your app package. For each webpage you want included as a tab in your app, you define a URL and a scope. Additionally, you need to add the [Teams JavaScript client SDK](/javascript/api/overview/msteams-client) to your page, and call `microsoftTeams.initialize()` after your page loads. Doing so will tell Teams to display your page, give you access to Teams-specific information (for example if the Teams client is running the *dark theme*), and allow you to take action based on the results.
```
Comment (Rajeshwari-v) [inline msteams-platform/tabs/what-are-tabs.md:42]: Rephrase the sentences to active, without changing the meaning

File: msteams-platform/tabs/what-are-tabs.md [hunk 6]
```diff
@@ -35,15 +35,15 @@ There are two types of tabs available in Teams — channel/group and personal. C
...
 
-Whether you choose to expose your tab within the channel/group or personal scope, you'll need to present an <iframe\> HTML [content page](~/tabs/how-to/create-tab-pages/content-page.md) in your tab. For personal tabs, the content URL is set directly in your Teams app manifest by the `contentUrl` property in the `staticTabs` array. Your tab's content will be the same for all users.
+Whether you choose to expose your tab within the channel or group or personal scope, you'll need to present an <iframe\> HTML [content page](~/tabs/how-to/create-tab-pages/content-page.md) in your tab. For personal tabs, the content URL is set directly in your Teams app manifest by the `contentUrl` property in the `staticTabs` array. Your tab's content will be the same for all users.
```
Comment (Rajeshwari-v) [inline msteams-platform/tabs/what-are-tabs.md:44]: you'll need to present an -
 better way of representation: you must present

File: msteams-platform/tabs/what-are-tabs.md [hunk 7]
```diff
@@ -35,15 +35,15 @@ There are two types of tabs available in Teams — channel/group and personal. C
...
+For channel or group tabs, you also need to create an additional configuration page that allows users to configure your content page URL, typically by using URL query string parameters to load the appropriate content for that context. This is because your channel or group tab can be added to multiple different teams or group chats. On each subsequent install, your users will be able to configure the tab, allowing you to tailor the experience as needed. When users add or configure a tab, a URL is being associated with the tab that is presented in the Teams UI. Configuring a tab is simply adding additional parameters to that URL. For example, when you add the Azure Boards tab, the configuration page allows you to choose which board the tab will load. The configuration page URL is specified by the  `configurationUrl` property in the `configurableTabs` array in your app manifest.
```
Comment (Rajeshwari-v) [inline msteams-platform/tabs/what-are-tabs.md:46]: Rephrase the section.
Eg: your users will be able to configure the tab
Better way: users can configure the tab

**Comment by v-abirade** [comment]
Created by mistake.

**Pull request closed without merging**